templates/form_theme.html.twig

```diff
@@ -170,3 +188,11 @@
         </div>
     </label>
 {% endblock %}
+
+{%- block checkbox_row -%}
```

Comment:
YOu are 100% sure it's not breaking anything ? Like rendering 2 times `<div class="form__checkbox">` in some places ?

Reply:
No, because checkbox_row is never called.
I can replace 
line 16
                {{- form_row(child, {translation_domain: choice_translation_domain}) -}}

by
    <div class="form__checkbox">
        {{ form_widget(form) }}
        {{ form_label(form) }}
        {{ form_errors(form) }}
    </div>